modules/core/test/de/diedavids/cuba/dataimport/binding/DatatypeFactoryIntegrationTest.groovy

```diff
@@ -112,6 +112,26 @@ class DatatypeFactoryIntegrationTest extends AbstractEntityBinderIntegrationTest
 
     }
 
+    @Test
+    void "getValue can parse a missing Date value correctly"() {
+
+        // given:
+        importAttributeMapper = attributeMapperFor('birthday')
+        importConfiguration = importConfigurationFor('ddcdi$MlbPlayer', importAttributeMapper)
+        def dateFormat = "yyyy-MM-dd"
+        importConfiguration.dateFormat = dateFormat
+
+        // and:
+        def expectedBindingValue = null
+        def bindRequest = bindRequestFor(importConfiguration, importAttributeMapper, [birthday: expectedBindingValue])
+
+        // when:
+        def result = sut.getValue(bindRequest)
+
+        // then:
+        assertThat(result).isEqualTo(expectedBindingValue)
```

Comment:
I think here we can simplify a little. How about that: 

```suggestion
        assertThat(result).isNull()
```

and remove the `expectedBindingValue` attribute